src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,18 +49,51 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
-  <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants"/>
+    <EnumProperty Name="PlatformTarget" DisplayName="Platform Target"/>
+    <StringProperty Name="Prefer32Bit" DisplayName="Prefer 32Bit"/>
+    <StringProperty Name="AllowUnsafeBlocks"  Default="False"  DisplayName="Allow unsafe code"/>
+    <StringProperty Name="Optimize" DisplayName="Optimize"/>
+    <EnumProperty Name="WarningLevel" DisplayName="Warning Level"/>
+    <StringProperty Name="NoWarn" DisplayName="Supress Warning"/>
+    <BoolProperty Name="TreatWarningsAsErrors" Visible="True"  Default="False" Description="Treat warnings as errors" />
+    <StringProperty Name="OutputPath" DisplayName="Output Path"/>
+    <StringProperty Name="DocumentationFile" DisplayName="Documentation file"/>
+    <EnumProperty Name="GenerateSerializationAssemblies" DisplayName="Generate serialization assemblies">
```

Comment:
Doesn't seem to throw error if the values outside this, but think might be handy to add validation.